### PR TITLE
ci: fix get_listener_url() in case of REST

### PR DESCRIPTION
**Pull request by mwasilew**
Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

**Comment by mwasilew** [comment]
I fixed the tests and covered the docker-compose use case. IMHO it can be merged now.

File: Dockerfile [hunk 1]
```diff
@@ -15,6 +15,7 @@ RUN apt-get update -q=2 && \
         python3-djangorestframework \
         python3-djangorestframework-filters \
         python3-djangorestframework-extensions \
+		python3-future \
```
Comment (chaws) [inline Dockerfile:18]: I think this indentation is tabbed

File: requirements.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ django_filter>=2.0
 djangorestframework>=3.9.2
 djangorestframework-filters>=1.0.0.dev0
 drf-extensions
+future
```
Comment (chaws) [inline requirements.txt:11]: I knew this fix would be fixed in the future :P

**Comment by chaws** [comment]
Did you rebase against master? I see the commit about removing CODEOWNERS file in this PR as well, but they have different commit hashes if compared to the one in master.

**Comment by chaws** [comment]
LGTM

**Pull request merged**